config.go

```diff
@@ -280,9 +280,9 @@ func (c *Config) lndConnectParams() (string, lndclient.Network, string,
 func defaultConfig() *Config {
 	return &Config{
 		HTTPSListen: defaultHTTPSListen,
+		TLSCertPath: DefaultTLSCertPath,
```

Comment:
We should also update `doc/config-lnd-integrated.md` and perhaps explain the difference (and reasoning) between the `lnd 10009` (gRPC only) and the `litd 8443` (grpc-web, REST, gRPC proxy, UI) a bit more explicitly?
Also this change should be made very explicit in the release notes (not sure where to put the reminder for us).